packages/certs-v5/commands/certs/index.js

```diff
@@ -4,16 +4,16 @@ let co = require('co')
 let cli = require('heroku-cli-util')
 let _ = require('lodash')
 
-let endpoints = require('../../lib/endpoints.js').certsAndDomains
+let endpoints = require('../../lib/endpoints.js').all
 let displayTable = require('../../lib/display_table.js')
 
 function * run (context, heroku) {
-  let certsAndDomains = yield endpoints(context.app, heroku)
+  let certs = yield endpoints(context.app, heroku)
```

Comment:
Was the name of this just wrong in the previous version? Or are we never dealing with both certs and domains together anymore?

Reply:
Mostly "just wrong"-ish. `endpoints` used to be an alias for a function called `certsAndDomains` which returned an object that contained both certs and domains....except we never actually used the domains anywhere in the entire codebase.

So, I removed the `certsAndDomains` and updated the function calls to all just use the actual `endpoints` function since the result is the same but the intention is clearer.